FireRender.Maya.Src/FireRenderViewport.cpp

```diff
@@ -794,7 +799,7 @@ void FireRenderViewport::readFrameBuffer(FireMaya::StoredFrame* storedFrame)
 	}
 
 	// Read the frame buffer.
-	RenderRegion region(0, m_contextPtr->width() - 1, 0, m_contextPtr->height() - 1);
+	RenderRegion region(0, m_contextPtr->width() - 1, m_contextPtr->height() - 1, 0);
```

Comment:
Why were top and bottom values in this call switched?
RenderRegion(unsigned int l, unsigned int r, unsigned int t, unsigned int b)

Reply:
It was error here, the fix is correct we just did not find it after I refactored switch functionality.
In Maya top and bottom is reversed, so top > bottom